### ISSUE TITLE
Basic graphs for Physical Track

### ISSUE THREAD

**Issue by andkov** [IALSA/IALSA-2015-Portland#96]
@ampiccinin , @GracielaMuniz , @annierobi ,  the first graph based on the results.csv has been produced (Screen below). You can see all graphs for the available pairs in [`./reports/physical/basic_graphs/figure_rmd`](https://github.com/IALSA/IALSA-2015-Portland/tree/master/reports/physical/basic_graphs/figure_rmd) or as a [md](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/basic_graphs/basic_graphs.md) or [html](http://htmlpreview.github.io/?https://raw.githubusercontent.com/IALSA/IALSA-2015-Portland/master/reports/physical/basic_graphs/basic_graphs.html) document. 
 
![gait_grip-1](https://cloud.githubusercontent.com/assets/2608073/10686299/855d64b4-7917-11e5-8612-6ad34e499038.png)

**Comment by andkov** [comment]
New report give the scatter plots of the factor scores extracted from .gh5 files. All graphs are labeled and stored in [`./reports/physical/fscores_scatter/figure_rmd`](https://github.com/IALSA/IALSA-2015-Portland/tree/master/reports/physical/fscores_scatter/figure_rmd) folder. You can view them as [md](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/fscores_scatter/fscores_scatter.md) or [html](htmlpreview.github.io/?https://raw.githubusercontent.com/IALSA/IALSA-2015-Portland/master/reports/physical/fscores_scatter/fscores_scatter.html) stored on github. 

![satsa_1-1](https://cloud.githubusercontent.com/assets/2608073/10706979/5a65f61a-79a1-11e5-86b3-072c84dc5f4f.png)

**Comment by andkov** [comment]
@ampiccinin , the graphs for all available . gh5 files has been uploaded. See this [README](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/fscores_scatter/figure_rmd/README.md) to view them online, however, I recommend syncing and viewing their local versions at [`./reports/physical/fscores_scatter/figure_rmd`](https://github.com/IALSA/IALSA-2015-Portland/tree/master/reports/physical/fscores_scatter/figure_rmd)

At this moment, the graphs depict the following models:
```
 [1] "./studies/eas/physical/b1_female_aehplus_grip_gait.gh5"  
 [2] "./studies/eas/physical/b1_female_aehplus_grip_pef.gh5"   
 [3] "./studies/eas/physical/b1_female_aehplus_pef_gait.gh5"   
 [4] "./studies/eas/physical/b1_male_aehplus_grip_gait.gh5"    
 [5] "./studies/eas/physical/b1_male_aehplus_grip_pef.gh5"     
 [6] "./studies/eas/physical/b1_male_aehplus_pef_gait.gh5"    
 
 [7] "./studies/hrs/physical/b1_female_aehplus_grip_gait.gh5"  
 [8] "./studies/hrs/physical/b1_female_aehplus_grip_pef.gh5"   
 [9] "./studies/hrs/physical/b1_female_aehplus_pef_gait.gh5"   
[10] "./studies/hrs/physical/b1_male_aehplus_grip_gait.gh5"    
[11] "./studies/hrs/physical/b1_male_aehplus_grip_pef.gh5"     
[12] "./studies/hrs/physical/b1_male_aehplus_pef_gait.gh5"     

[13] "./studies/lasa/physical/b1_female_aehplus_gait_grip.gh5" 
[14] "./studies/lasa/physical/b1_female_aehplus_pek_gait.gh5"  
[15] "./studies/lasa/physical/b1_female_aehplus_pek_grip.gh5"  
[16] "./studies/lasa/physical/b1_male_aehplus_gait_grip.gh5"   
[17] "./studies/lasa/physical/b1_male_aehplus_pek_gait.gh5"    
[18] "./studies/lasa/physical/b1_male_aehplus_pek_grip.gh5"    

[19] "./studies/octo/physical/b1_female_aehplus_gait_grip.gh5" 
[20] "./studies/octo/physical/b1_female_aehplus_pek_gait.gh5"  
[21] "./studies/octo/physical/b1_female_aehplus_pek_grip.gh5"  
[22] "./studies/octo/physical/b1_male_aehplus_gait_grip.gh5"   
[23] "./studies/octo/physical/b1_male_aehplus_pek_gait.gh5"    
[24] "./studies/octo/physical/b1_male_aehplus_pek_grip.gh5"   
 
[25] "./studies/radc/physical/b1_female_aehplus_fev_gait.gh5"  
[26] "./studies/radc/physical/b1_female_aehplus_fev_grip.gh5"  
[27] "./studies/radc/physical/b1_female_aehplus_gait_grip.gh5" 
[28] "./studies/radc/physical/b1_male_aehplus_fev_gait.gh5"    
[29] "./studies/radc/physical/b1_male_aehplus_fev_grip.gh5"    
[30] "./studies/radc/physical/b1_male_aehplus_gait_grip.gh5"   

[31] "./studies/satsa/physical/b1_female_aehplus_gait_fev.gh5" 
[32] "./studies/satsa/physical/b1_female_aehplus_gait_grip.gh5"
[33] "./studies/satsa/physical/b1_female_aehplus_grip_fev.gh5" 
[34] "./studies/satsa/physical/b1_male_aehplus_gait_fev.gh5"   
[35] "./studies/satsa/physical/b1_male_aehplus_gait_grip.gh5"  
[36] "./studies/satsa/physical/b1_male_aehplus_grip_fev.gh5" 
```

**Comment by ampiccinin** [comment]
@andkov - Wow! Thanks!

Quick Q – the axes seem quite different across some plots – e.g., OCTO Female gait-grip vs pek-gait.

e.g., gait: -0.1 to -0.5 in plot with grip,
                    10 to -20 with pek, and one outlier beyond 30.

Maybe I just need more sleep…

A

**Comment by ialsa-lab** [comment]
Yes, many room for improvement is there. Keep dropping your ideas, I’ll try to implement them to the best of my ability.

**Comment by andkov** [comment]
@ampiccinin , new graphs are ready that compare observed values (smoothed) and values predicted from the factor scores (fscores).  View by [process pair](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/fscores_scatter/figure_rmd/by_process_pair.md) or by [study](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/fscores_scatter/figure_rmd/by_study.md). Some of the labels are mixed up: if you pay attention to scales you'll see. A few people didn't follow convention that I have [proposed](https://github.com/IALSA/IALSA-2015-Portland/blob/master/projects/physical/announce.md) so we ended up with `grip-gait` as well as `gait-grip` models. I didn't really expect that go perfectly anyway, but haven't found a way to sort them out automatically yet.  For most part it doesn't matter, but in cases like these - it does. So knowing the scale on which a measure is captures is useful here. 

P.S. As an idea, now any graph can be referenced by a hyperlink. I suggest we start using Issues as discussion board by providing a graph an a description of it that supports our story about it. 

![radc_female_aehplus_gait_grip-1](https://cloud.githubusercontent.com/assets/2608073/10714777/1ba9ded0-7abb-11e5-8309-65d16a697150.png)
.md)

**Comment by ampiccinin** [comment]
Thanks @andkov. For some reason I can’t see them.

(also – would it be better if the lower left graphs were IC-SC? What was your reasoning for IP-SC? Thanks!)

**Comment by andkov** [comment]
@ampiccinin , are you saying that the following links ( view by [process pair](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/observed_predicted/figure_rmd/by_process_pair.md) or by [study](https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/observed_predicted/figure_rmd/by_study.md)) do not open in your browser? 


Yes, you are right . That would be logical. I think i just saw some other symmetry. I agree, having IC-SC in lower left is the proper configuration. I'll correct it on the next pass

**Comment by ampiccinin** [comment]
... I was, but now they do - thanks! Cool!

On Oct 25, 2015, at 1:53 PM, Andriy V. Koval <notifications@github.com<mailto:notifications@github.com>> wrote:


@ampiccinin<https://github.com/ampiccinin> , are you saying that the following links ( view by process pair<https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/observed_predicted/figure_rmd/by_process_pair.md> or by study<https://github.com/IALSA/IALSA-2015-Portland/blob/master/reports/physical/observed_predicted/figure_rmd/by_study.md>) do not open in your browser?

Yes, you are right . That would be logical. I think i just saw some other symmetry. I agree, having IC-SC in lower left is the proper configuration. I'll correct it on the next pass

-

**Comment by chenkaiwu** [comment]
Hi Andrey,

 

Could you help check why gait speed against peak flow grpahs for the HRS are not available on https://github.com/IALSA/IALSA-2015-Portland/tree/master/reports/physical/fscores_scatter/figure_rmd? Thanks. 

 

Chenkai

**Comment by ampiccinin** [comment]
☺ - sorry! I had intended to cc: Andrey when I wrote that! Was interrupted and apparently forgot!! A
